ecl/hql/hqlexpr.hpp

```diff
@@ -726,6 +726,7 @@ enum node_operator : unsigned short {
         no_merge_pending,
         no_httpcall,
         no_getenv,
+        no_getsecret,
```

Comment:
See comment from before - this needs to be before no_last_op, or replace a no_unused<n>